client.go

```diff
@@ -82,3 +84,10 @@ func (c *Client) doRequest(method string, path string, params url.Values, body i
 
 	return c.httpClient.Do(req)
 }
+
+// Signalflow returns a Signalflow client that can be used to execute streaming
+// jobs.
+func (c *Client) Signalflow(options ...signalflow.ClientParam) (*signalflow.Client, error) {
```

Comment:
Should this be `SignalFlow` because Marketing™?

Reply:
Yes, changed it to the proper spelling.